### ISSUE TITLE
Not working in Android 8.1? 

### ISSUE THREAD

**Issue by tarroyo8** [blokadaorg/blokada#204]
### Steps to reproduce
1. Install Blokada app
2. Open Blokada
3. Tap the button to power it on, When the connection request appears I'm unable to tap 'ok' Nothing happens when clicking it.

### Expected behaviour

I would expect Blokada to turn on with no issues.

### Actual behaviour

It's like the app freezes. I am able to tap cancel only. I tried force stopping it, reinstalling, clearing cache and data the same issue occurs.

### Configuration
1. Blokada version 3.3

2. Device model
Google Pixel 2 XL

3. Android version / ROM
8.1/ Stock - Build Number OPM2.171019.029 (April Security Patch)

### Questions
1. How long have you been using Blokada? 
I installed it today and had this happen right away.

2. How long time has the problem started to happen? See above

3.  How often does this problem happen to you? Since installing it.

**Comment by rpeter85** [comment]
Blokada works with Android 8.1, that's what I also have.
To get more info about the issue, do you have other, active VPN connection? Did you restart the phone after you installed Blokada? Sounds odd, but sometimes it solves the issue.

**Comment by tarroyo8** [comment]
No, No other VPN is active. It's weird I tried it on my other phone which runs 8.0 and had no problems getting it to power on.

**Comment by rpeter85** [comment]
Collect the logs (either from the top right corner or using logcat), otherwise it will be just a guessing game.

**Comment by tarroyo8** [comment]
U should be getting one now, I just sent one.

**Comment by tarroyo8** [comment]
Check the link I used above, It's not a dimming app it's a screen balance app that you can manually adjust colors. That's what seemed to be causing this issue

**Comment by kar** [comment]
Ah yeah indeed, I haven't noticed that comment.

@rpeter85 it'd be nice to add a question about apps drawing overlays to Help. Such apps will prevent Android from allowing to accept certain permission dialogs, like the VPN dialog. It's for protecting users from malware apps auto-granting permissions for themselves.

**Comment by e40** [comment]
@rpeter85 so, basically 1Password on Android.  That's about when Blokada stopped working for me.  This is a disappointing state of affairs.  Either 1Password or Blokada, but not both!  Bummer!

**Comment by e40** [comment]
How can we check if there are apps with overlays?

**Comment by ahmadsamhann** [comment]
@e40 check the app's settings. draw on other apps IIRC. 

_Sent from my Mi A1 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by e40** [comment]
@ahmadsamhann I found that and disabled all of them, Blokada still doesn't work (stop/starts, repeat).

**Comment by e40** [comment]
I really think this should be reopened.

**Comment by e40** [comment]
OK, found the culprit: Google's Datally.  App to help not use mobile data.  Uninstalling that made Blokada work again.
